bot/exts/info/resources.py

```diff
@@ -0,0 +1,79 @@
+    # Obtain the topics using CSS selector
+    topics_initial = soup.select('label.checkbox input[data-filter-name="topics"]')
+    # Split topics at the dash, add to list, prep for iteration over each word
+    for topic in topics_initial:
+        topics_split.append(topic['data-filter-item'].split("-"))
+    # Iterate over each word in order to title case and remove the OTHER category (not helpful to include in embed)
+    for topic in topics_split:
+        for word in range(0, len(topic)):
+            if topic[word] != "other":
+                if topic[word] not in dont_capitalize:
+                    topics_title += topic[word].title() + " "
+                if topic[word] in dont_capitalize:
+                    topics_title += topic[word] + " "
+                if word == len(topic) - 1:
+                    SITE_TOPICS_TITLE_CASE.append(topics_title)
+                    topics_title = ""
+
+
+class Resources(commands.Cog):
+    """Displays information about the Python Discord website Resource page."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(name="resources", aliases=("res",))
+    async def resources_command(self, ctx: commands.Context, *, resource_topic: Optional[str]) -> None:
+        """Display information and a link to the Python Discord website Resources page."""
+        # If resource_topic isn't provided, provide link to the resources' page without any filtering of topics
+        if not resource_topic:
+            embed = Embed(
+                title="Resources",
+                description="The [Resources page](https://www.pythondiscord.com/resources) on our "
+                            "website contains a list of hand-selected learning resources that we "
+                            "regularly recommend to both beginners and experts."
+                            "\n\nLearning resource topics include:"
+            )
+            # Display the current topics on the Python Discord website to better inform the user of what's available
+            get_topics()
+            for topic in SITE_TOPICS_TITLE_CASE:
+                embed.add_field(name=f"{topic}", value="\u200b")
+            await ctx.send(embed=embed)
+            SITE_TOPICS_TITLE_CASE.clear()
+            return
+        # Filter the topics by replacing spaces with dashes for use in URL
+        if len(resource_topic) > 1:
+            resource_topic = resource_topic.replace(" ", "-")
+        embed = Embed(
+            title="Resources",
+            description=f"The "
+                        f"[Resources page](https://www.pythondiscord.com/resources/?topics={resource_topic}) "
+                        f"on our website contains a list of hand-selected learning resources that we "
+                        f"regularly recommend to both beginners and experts."
+        )
+        await ctx.send(embed=embed)
+
+
+def setup(bot: Bot) -> None:
+    """Load the BotSource cog."""
```

Comment:
Don't forget to update this docstring!
```suggestion
    """Load the Resources cog."""
```